config/reports.prd.json

```diff
@@ -0,0 +1,309 @@
+{
+    "layout": [
+        {
+            "type": "diamondash.widgets.lvalue.LValueWidget",
+            "time_range": "30d",
+            "name": "Total Unique Users",
+            "target": {
+                "metric_type": "account",
+                "store": "usaid_clinicfinder_prd",
+                "name": "sum.unique_users",
+                "aggregator": "last"
+                }
+        },
+
+        {
+            "type": "diamondash.widgets.graph.GraphWidget",
+            "name": "Unique Users Per Week",
+            "time_range": "112d",
+            "bucket_size": "7d",
+            "width": 6,
+            "metrics": [
+                {
+                    "name": "Unique Users",
+                    "target": {
+                        "metric_type": "account",
+                        "store": "usaid_clinicfinder_prd",
+                        "name": "sum.unique_users",
+                        "aggregator": "last"
+                    }
+                }
+            ]
+        },
+
+
+        "new_row",
+
+        {
+            "type": "diamondash.widgets.graph.GraphWidget",
+            "name": "New vs Returning Users",
+            "time_range": "365d",
+            "bucket_size": "1d",
+            "width": 6,
+            "metrics": [
+                {
+                    "name": "Found One Clinic Users",
```

Comment:
That means something else to me. Use `Total One Time Users` and `Total Returning Users`